x-pack/plugins/encrypted_saved_objects/server/plugin.ts

```diff
@@ -63,6 +64,12 @@ export class EncryptedSavedObjectsPlugin
         'Saved objects encryption key is not set. This will severely limit Kibana functionality. ' +
           'Please set xpack.encryptedSavedObjects.encryptionKey in the kibana.yml or use the bin/kibana-encryption-keys command.'
       );
+    } else {
+      const hashedEncryptionKey = createHash('sha3-256')
+        .update(config.encryptionKey)
+        .digest('base64');
+
+      this.logger.info(`Hashed 'encryptionKey' for this instance: ${hashedEncryptionKey}`);
```

Comment:
Just a nit/question - do we want to refer to the yml field path (xpack.encryptedSavedObjects.encryptionKey) just for clarity of where the setting is located?

Reply:
++ you can technically figure this out by reading which plugin wrote the log message, but I think spelling out the entire path would be easier to parse (and search for in logs!)

Reply:
Good call! Added in latest commit!